### PR TITLE
Update UI recipes to use `@storybook/addon-styling`

### PR DESCRIPTION
**Pull request by Integrayshaun**
## What changed
- [x] Update Tailwind recipe
- [x] Update Material UI recipe
- [x] Update Styled Components recipe
- [x] Update Emotion recipe

### CODE REVIEW

**Comment by Integrayshaun** [comment]
Adding new icon

bootstrap

File: src/content/recipes/@emotion/styled.md [hunk 1]
```diff
@@ -321,76 +318,28 @@ Now, to get the most out of your stories, there should be a way to toggle betwee
...
-
-  return (
-    <ThemeProvider theme={THEMES[theme] || THEMES['light']}>
-      <GlobalStyles />
-      <Story />
-    </ThemeProvider>
-  );
-};
-```
+Adding a second theme will create a new toolbar menu to select your desired theme for your stories.
```
Comment (kylegach) [inline src/content/recipes/@emotion/styled.md:342]: 🤩 Now _that's_ some intuitive DX.

File: src/content/recipes/@mui/material.md [hunk 2]
```diff
@@ -75,90 +87,44 @@ export const darkTheme = createTheme({
...
-  },
+```js
+module.exports = {
+  stories: ['../stories/**/*.stories.mdx', '../stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-essentials', '@storybook/addon-styling'],
 };
 ```
 
-Now we can update our decorator to provide the theme selected in our new dropdown.
+And finally apply the custom themes to our stories, we’ll need to wrap them in Material UI’s `ThemeProvider` using the `withThemeFromJSXProvider` decorator.
```
Comment (kylegach) [inline src/content/recipes/@mui/material.md:105]:
```suggestion
And finally apply the custom themes to our stories. We’ll need to wrap them in Material UI’s `ThemeProvider` using the `withThemeFromJSXProvider` decorator.
```

File: src/content/recipes/@emotion/styled.md [hunk 3]
```diff
@@ -29,33 +29,51 @@ If you’d like to see the example code of this recipe, check out the [example r
...
+        font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+      }
+    `}
+  />
 );
 
-export const decorators = [withGlobalStyle];
+export const decorators = [
+  withThemeFromJSXProvider({
+    GlobalStyles: GlobalStyle, // Adds your GlobalStyle component to all stories
```
Comment (kylegach) [inline src/content/recipes/@emotion/styled.md:74]: If you name the `GlobalStyle` component `GlobalStyles` instead, you can simplify this line to:

```suggestion
    GlobalStyles, // Adds your GlobalStyle component to all stories
```

File: src/content/recipes/tailwindcss.md [hunk 4]
```diff
@@ -55,44 +55,37 @@ module.exports = {
...
-+   });
-+   // Return the altered config
-+   return config;
-+ },
-}
+  // snipped for brevity
+};
 ```
 
+**Note**: Using Vite, `@storybook/nextjs`, or `@storybook/preset-create-react-app` with `react-scripts@2.0.0` and up? You don't need to set `postCss` to true.
```
Comment (kylegach) [inline src/content/recipes/tailwindcss.md:79]: ❤️

File: src/content/recipes/tailwindcss.md [hunk 5]
```diff
@@ -139,67 +132,28 @@ module.exports = {
...
+    attributeName: 'data-mode',
+  }),
+];
 ```
 
+This code will create a new toolbar menu to select your desired theme for your stories.
+
 ## Get involved
 
 Now you're ready to use Tailwind with Storybook. 🎉 Check out the [example repo](https://github.com/Integrayshaun/storybook-tailwind-recipe-example) for a quick start.
```
Comment (kylegach) [inline src/content/recipes/tailwindcss.md:159]: _I can't comment on `bootstrap.svg`, so I'm doing so here._

Wrong PR for this file to be added?

**Review by kylegach — APPROVED** [review]
Lovely! Just some minor suggestions.

**Pull request merged**